plugin.video.vrt.nu/addon.xml

```diff
@@ -16,9 +16,7 @@
     <extension point="xbmc.python.pluginsource" library="addon.py">
         <provides>video</provides>
     </extension>
-
-    <extension point="xbmc.service" library="service.py" start="startup" />
```

Comment:
Do we remove `start="startup"` here on purpose ?

Reply:
Yes, cause this option doesn't exist anymore in Kodi Leia: look at the bottom of the page at https://kodi.wiki/view/Service_add-ons

Reply:
Ok, but it does for Krypton ? Because we still release for Krypton and Leia.

Reply:
Ok, so I guess we don't need it even when it meant something ?

Reply:
Correct, we don't need it. Everything works as expected without it.